gazebo/common/Profiler.hh

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2018 Open Source Robotics Foundation
```

Comment:
`2018` -> `2020`

Reply:
I was leaving them with the original date since they were copied from `ign-common`. I'll add a note to each of them about where they came from, what do you think?

Reply:
I see, yeah adding a note sounds good

Reply:
0afd239